src/main/java/uk/gov/hmcts/darts/arm/mapper/impl/MediaArchiveRecordMapperImpl.java

```diff
@@ -128,7 +128,9 @@ private RecordMetadata createArchiveRecordMetadata(ExternalObjectDirectoryEntity
             .publisher(armDataManagementConfiguration.getPublisher())
             .recordClass(armDataManagementConfiguration.getMediaRecordClass())
             .recordDate(formatDateTime(currentTimeHelper.currentOffsetDateTime()))
-            .eventDate(formatDateTime(media.getCreatedDateTime()))
+            .eventDate(formatDateTime(nonNull(media.getRetainUntilTs())
+                                          ? media.getRetainUntilTs().minusYears(armDataManagementConfiguration.getEventDateAdjustmentYears())
```

Comment:
similar to previous comment - should getRetainUntilTs be renamed getRetainUntilDateTime? (even though I don't think you created it here)

Reply:
Renaming shouldn't be part of this ticket as this will impact other areas like automated job to calculate retention dt etc.